### PR TITLE
Replace captcha with recaptcha for mweb

### PR DESCRIPTION
**Pull request by jackniu1**
👓 @uzi @KeyserSosa 

Have to wait for below PR deployed first:

https://github.com/reddit/reddit-public/pull/5828

### CODE REVIEW

File: src/apis/PostsEndpoint.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ const formatPostData = (data)=> {
     sendreplies: data.sendreplies,
     sr: data.sr,
     iden: data.iden,
-    captcha: data.captcha,
+    'g-recaptcha-response': data.gRecaptchaResponse,
```
Comment (uzi) [inline src/apis/PostsEndpoint.js]: What is `iden` and do we still need it after switching to recaptcha?

Comment (jackniu1) [inline src/apis/PostsEndpoint.js]: Yep, it was used to identify whether the recaptcha need show or not. Will send another PR of reddit-mobile soon.

**Comment by KeyserSosa** [comment]
:fish: assuming I shouldn't be worried about the unit test!

**Comment by uzi** [comment]
:fish: as well ... but of course have to wait for that other patch
